client-ts/signalr/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aspnet/signalr",
-  "version": "1.0.0-preview1-t000",
+  "version": "1.0.0-preview1-update1",
```

Comment:
I'm going to undo these changes and just do them locally when building. The CI will be replacing the version with a build number anyway.